paf-mvp-benchmark/K6/first-visit-with-no-3pc-safari.js

```diff
@@ -0,0 +1,55 @@
+import {getOptions, getCMPHost, getBaseHost, getSleepDuration } from './config.js';
```

Comment:
I don't understand what's the purpose of this specific scenario, it looks very similar to the next one.
Can you explain?

Reply:
It's temporary to check the effect of User-Agent. Will be removed if no effect.